### ISSUE TITLE
Custom Items: Errors and Unworkable

### ISSUE THREAD

**Issue by Aternix** [zhuowei/MCPELauncher#1039]
Sometimes some mods work in 0.14x but sometimes not. I tell you why:

Making custom item is "ModPE.setItem("asdf", 0, "asdf", 0)

And after that importing... Shows "custom item asdf:0 not found"

Please update it again and supports custom items in JS files and also custom textures too. Please response in 30 days.

**Comment by Black-DeathPE** [comment]
It does work.

**Comment by Aternix** [comment]
Im trying to importing effect sword but its still shows errors.

Explain to me and why they are working from you?

Do you use modpkg file? This is only supported for me.
Custom items without additional texture will works.

**Comment by Aternix** [comment]
Hmm. I seeit. Var things is needed to work my mod!

**Comment by LawRence-G-38** [comment]
Maybe because MCPE `0.14.2` and above has codes of `0.15.0 Realms` so `0.14.2` is different than `0.14.1` and `0.14.0`.

**Comment by AJ12345678910** [comment]
I'm having trouble getting BlockLauncher to work on MCPE 0.14.3 too.

**Comment by Aternix** [comment]
They need variable things in javascript!

**Comment by Aternix** [comment]
Nope... 0.14.0-3 mods can use 0.14.0 mod can use in 0.14.1, 0.14.1 mod can
use in 0.14.2 and more...

Because custom items/blocks and texture names must be with variable things.
A texture can use with custom items in item-opaque folder like
earth_sword_0.png reads by Javascript.

**Comment by Aternix** [comment]
0.15.0 beta isn't supported with mods when using blocklauncher and
mcpelauncher. Only 0.15.0 alpha builds works.
